engine/graphic/src/Texture.cpp

```diff
@@ -0,0 +1,86 @@
+#include "Texture.h"
+Texture::Texture(char* FileName)
+{
+	m_fileName = FileName;
+	imageId = 0;
```

Comment:
Maybe it will be better to set ID for texture manually?
In case of a lot of texture atlases it will be very convenient